source/creating_doc/doc_home.rst

```diff
@@ -2,28 +2,48 @@ Open edX Documentors
 =============================
 
 
-**Creating Documentation**
+.. panels::
 
-:doc:`quick_start_build_doc`
+   **Quick Starts**
 
-:doc:`doc_guidelines`
+   :doc:`quick_starts/quick_start_build_doc`
+   :doc:`quick_starts/update_doc_via_github`
+   :doc:`quick_starts/quick_start_rst`
 
-:doc:`doc_templates`
+   .. link-button:: quick_starts
+    :type: ref
+    :text: All Documentor Quick Starts
+    :classes: btn-outline-primary btn-block
 
-:doc:`doc_checklist`
+   ---
 
-:doc:`readme`
+   **Concepts**
 
-:doc:`quickstart_rst`
+   :doc:`concepts/doc_audiences`
+   :doc:`concepts/content_types`
 
+   .. link-button:: concepts
+    :type: ref
+    :text: All Documentor Concepts
+    :classes: btn-outline-primary btn-block
 
-.. toctree::
-   :glob:
-   :caption: HOW-TOs
+   ___
 
-   how-tos/*
+   **References**
 
+   :doc:`references/doc_guidelines`
```

Comment:
These need bullets or a blank line between them because otherwise multiple links are showing up on the same line without any clear separation.